.vsts.ci.yml

```diff
@@ -36,20 +36,92 @@ phases:
 
   # Upload agent package zip as build artifact
   - task: PublishBuildArtifacts@1
-    displayName: Publish Artifact (Windows)
+    displayName: Publish Artifact (Windows x64)
+    condition: and(succeeded(), ne(variables['build.reason'], 'PullRequest'))
+    inputs:
+      pathToPublish: _package
+      artifactName: agent
+      artifactType: container
+
+################################################################################
+- phase: build_windows_x86_agent
+################################################################################
+  displayName: Windows Agent (x86)
+  queue:
+    name: buildDevs
```

Comment:
Minor: Is it intentional that the name attribute for the x86 agent is `buildDevs`, but `Hosted VS2017` for the x64 agent?

Reply:
yes. the hosted agent is Azure is 64 bit, i have to create a 32 bits build machine in the BuildDevs pool

Reply:
Yep, I now recall the conversation.